krkn/scenario_plugins/application_outage/application_outage_scenario_plugin.py

```diff
@@ -122,3 +146,48 @@ def rollback_network_policy(
 
     def get_scenario_types(self) -> list[str]:
         return ["application_outages_scenarios"]
+
+    @staticmethod
+    def _build_exclude_expressions(exclude_label) -> list[dict]:
```

Comment:
LGTM, my two cents, why don't you move this method in the utils package and you make it public?  Might be absolutely reusable for something else.

Reply:
you already have a functions.py that should be enough, for the moment I don't see the need of a new file for a single method, let's keep them here and then, if in the future we'll have multiple label related methods, we'll refactor. Thanks again!